core/network/src/main/java/com/android/developers/androidify/RemoteConfigDataSource.kt

```diff
@@ -106,4 +108,8 @@ class RemoteConfigDataSourceImpl @Inject constructor() : RemoteConfigDataSource
     override fun getBotBackgroundInstructionPrompt(): String {
         return remoteConfig.getString("bot_background_instruction_prompt")
     }
+
+    override fun isXrDisabled(): Boolean {
+        return remoteConfig.getBoolean("is_xr_disabled")
```

Comment:
I've added a parameter to Firebase Remote Config in the same naming convention as wearOS watchfaces : `xr_feature_enabled` - at the moment its defaulting to false.

Reply:
Added 6f47435d2e4f993e0da75a5ef8d34683363d791c which reflects this!